capstone/src/main/java/com/google/univiz/api/dataviz/CollegeStats.java

```diff
@@ -0,0 +1,42 @@
+package com.google.univiz.api.dataviz;
+
+import com.google.auto.value.AutoValue;
+
+/** CollegeStats is a class that contains statistics related to tuition for a particular college. */
+@AutoValue
```

Comment:
nit - This AutoValue will be shipped to the client - so it will make sense to add a `@GenerateTypeAdapter`...